packages/express/index.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  get Mixin() {
+    return require('./mixin.core');
+  },
+  get uri() {
+    return require('./lib/uri');
+  },
+  get createServer() {
+    return require('@untool/core').bootstrap().createServer;
```

Comment:
Does this work without having to call `handleArguments({})`? In an earlier version of untool we had to do this in order to set up logging correctly (I think it crashed - or just didn't log anything?)

Reply:
True, it does not log anything unless you also call `handleArguments()`. We will probably also have to expose that - or would you consider it appropriate to call it here ourselves?

Reply:
I think we ought not: `handleArguments` is not just being used to set the log level.

Looking at what [untool/webpack](https://github.com/untool/untool/blob/f21a94533edd8fbd741c16440ddbc7d00ba69408/packages/webpack/mixin.core.js#L220)) is doing, I am not entirely sure if makes a lot of sense to expose these hooks here at all. Will think about it.